src/channel.js

```diff
@@ -1,8 +1,12 @@
 const innerEvents = [
   'onListenerAdded',
   'onListenerRemoved',
+  'onProxyChannelAdded',
```

Comment:
All of these is not needed:
'onProxyChannelAdded',
'onProxyChannelRemoved',
'onFirstProxyChannelAdded',
'onLastProxyChannelRemoved'

Proxying is just adding special listener.
Please update pr.